js/templates/pageNav.html

```diff
@@ -8,7 +8,7 @@ <h3 class="margin0 marginTop25"><%= polyglot.t('Discover') %></h3>
   </a>
 </div>
 
-<div class="popMenu hide popMenu-navBar js-navProfileMenu">
+<div class="popMenu popMenu-navBar js-navProfileMenu">
```

Comment:
Ok, this should be fixed.

Totally forgot about translateY, which does exactly what we need (percentage relative to the element, not container).